packages/cli/package.json

```diff
@@ -19,12 +19,12 @@
     "/oclif.manifest.json"
   ],
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
```

Comment:
here's the "real" breaking change 😉 so we should rebase these changes into `zapier-platform-major-release-16.0.0-dev` instead of `main`

Reply:
Done!